src/vms/context/model.ts

```diff
@@ -9,14 +9,8 @@ export type Context = {
   readonly avaxAssetID: string;
   readonly baseTxFee: bigint;
   readonly createAssetTxFee: bigint;
-  readonly createSubnetTxFee: bigint;
-  readonly transformSubnetTxFee: bigint;
-  readonly createBlockchainTxFee: bigint;
-  readonly addPrimaryNetworkValidatorFee: bigint;
-  readonly addPrimaryNetworkDelegatorFee: bigint;
-  readonly addSubnetValidatorFee: bigint;
-  readonly addSubnetDelegatorFee: bigint;
 
   // Post Etna
   readonly platformFeeConfig: FeeConfig;
+  readonly feeState: FeeState;
```

Comment:
I don't think it's appropriate to attach the `FeeState` to the context. The context is pretty static -- you fetch it once. The `FeeState` changes from block to block and has to be refetched for every tx...